lib/faraday.rb

```diff
@@ -19,7 +19,7 @@
 #   conn.get '/'
 #
 module Faraday
-  VERSION = '0.16.0'
+  VERSION = '0.16.1'
```

Comment:
Please let the maintenance team do the update to this file.

Can you undo this change?

Reply:
Yep!